From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alberto Murillo <alberto.murillo.silva@intel.com>
Date: Thu, 17 Dec 2015 22:59:02 -0600
Subject: [PATCH] stateless

---
 configure.ac | 2 +-
 src/magic.c  | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 7da62aa..41d8cd1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -71,7 +71,7 @@ esac
 AC_SUBST(MINGW)
 AM_CONDITIONAL(MINGW, test "$MINGW" = 1)
 
-AC_SUBST([pkgdatadir], ['$(datadir)/misc'])
+AC_SUBST([pkgdatadir], ['$(datadir)/file'])
 AC_SUBST(fsect)
 AM_CONDITIONAL(FSECT5, test x$fsect = x5)
 
diff --git a/src/magic.c b/src/magic.c
index da5baf1..7e06c18 100644
--- a/src/magic.c
+++ b/src/magic.c
@@ -122,7 +122,7 @@ static void
 _w32_get_magic_relative_to(char **hmagicpath, HINSTANCE module)
 {
 	static const char *trypaths[] = {
-		"%s/share/misc/magic.mgc",
+		"%s/share/file/magic.mgc",
 		"%s/magic.mgc",
 	};
 	LPSTR dllpath;
@@ -146,7 +146,7 @@ _w32_get_magic_relative_to(char **hmagicpath, HINSTANCE module)
 	sp = strlen(dllpath);
 	if (sp > 3 && stricmp(&dllpath[sp - 3], "bin") == 0) {
 		_w32_append_path(hmagicpath,
-		    "%s/../share/misc/magic.mgc", dllpath);
+		    "%s/../share/file/magic.mgc", dllpath);
 		goto out;
 	}
 
